src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,25 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
+
+- For ASP.NET Core 6, .NET 6
+- For ASP.NET Core 5, .NET 5
+- For ASP.NET Core 3, .NET Core 3.1
+- For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
+
+Registration of NLog.Web in the NLog.config File
+	<!-- enable ASP.NET layout renderers -->
```

Comment:
Also notice that NLog.Web is for ASP.NET Classic, and NLog.Web.AspNetCore is for ASP.NET Core

Would it make sense to use code-markup as xml ?

Reply:
yes, done

Reply:
Thank you for the markup, but the xml-code still says `NLog.Web` and not `NLog.Web.AspNetCore`

Reply:
sorry about that, fixed